docs/playbook/getting-set-up/notebook-setup.md

```diff
@@ -0,0 +1,157 @@
+---
+title: Local Notebook Environment Setup
+description: "Local Notebook Environment Setup"
+layout: playbook_js
+tags: [playbook, setup]
+---
+
+## Introduction
+
+**What is the notebooking tool**
+
+Essentially a Jupyter server so that we can run Jupyter Notebooks. A Jupyter notebook is a web-based interactive development environment that runs Python code in a sequence of cells, which can be individually executed in the form of a notebook. This makes it easier to develop code and try things out on your own computer.
```

Comment:
Jupyter server won't mean anything to anyone new to the DP so I think delete the first sentence (Essentially a Jupyter server so that we can run Jupyter Notebooks.)